### PR TITLE
Switch to Logback

### PR DESCRIPTION
**Pull request by zax71**
Replaces sfl4j with Logback configured to colourise logs

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -24,7 +24,7 @@ dependencies {
     implementation("com.github.Minestom:Minestom:-SNAPSHOT")
 
     // Enable logging in Minestom
-    implementation("org.slf4j:slf4j-simple:2.0.13")
+    implementation("ch.qos.logback:logback-classic:1.4.14")
```
Comment (mudkipdev) [inline build.gradle.kts]: Latest version of logback-classic is 1.5.6

File: src/main/java/org/example/server/LogbackHighlightConfig.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package org.example.server;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.pattern.color.ANSIConstants;
+import ch.qos.logback.core.pattern.color.ForegroundCompositeConverterBase;
+
+public class LogbackHighlightConfig extends ForegroundCompositeConverterBase<ILoggingEvent> {
```
Comment (mudkipdev) [inline src/main/java/org/example/server/LogbackHighlightConfig.java]: Make this class final

File: src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+    <conversionRule conversionWord="highlightex" converterClass="org.example.server.LogbackHighlightConfig"/>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss} %highlightex([%level]): %highlightex(%msg%n)</pattern>
```
Comment (mudkipdev) [inline src/main/resources/logback.xml]: Time format should be `h:mm:ss a` to match what it was before (I use 24 hour time myself but I assumed there would be more people wanting 12 hour time)

**Review by mudkipdev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/server/LogbackHighlightConfig.java (hunk 2 above)
Comment (zax71) [reply src/main/java/org/example/server/LogbackHighlightConfig.java]: I never knew you could have final classes xD Tine to do some googling

**Pull request merged**